libraries/botbuilder-ai-orchestrator/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+      "target": "es6",
+      "lib": ["es2015", "dom"],
```

Comment:
Since the orchestrator library seems tied to the filesystem (and the declarative library definitely so), the `"dom"` lib seems unnecessary.

Based on the current appearance, the library doesn't seem like it should ever run in the browser, is this correct?